provider/datagen/tests/data/json/datetime/week_data@1/ar.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Thought: seems like we shouldn't be adding these.

Reply:
Agreed